src/parser/parse_tree.h

```diff
@@ -3502,10 +3504,7 @@ union pt_statement_info
   PT_TRACE_INFO trace;
   PT_KILLSTMT_INFO killstmt;
   PT_WITH_CLAUSE_INFO with_clause;
-  PT_ALTER_SYNONYM_INFO alter_synonym;
-  PT_CREATE_SYNONYM_INFO create_synonym;
-  PT_DROP_SYNONYM_INFO drop_synonym;
-  PT_RENAME_SYNONYM_INFO rename_synonym;
+  PT_SYNONYM_INFO synonym;
```

Comment:
It is recommended to move the location after the PT_SPEC_INFO spec.
Existing contents are sorted in ascending order by name.

Reply:
Changed the order.